Makefile

```diff
@@ -94,13 +100,33 @@ unit-test: build-dirs
 	./hack/unit-test.sh
 .PHONY: unit-test
 
+integration-test:
+	@go build ./test
+.PHONY: integration-test
+
 check:
 	@echo "Begin to check code formats."
 	./hack/check.sh
 	@echo "Begin to check dockerd whether is startup"
 	./hack/check-docker.sh
 .PHONY: check
 
+go-mod-tidy:
+	@echo "Begin to tidy up go.mod and go.sum"
+	@go mod tidy
+.PHONY: go-mod-tidy
+
+# we need this because currently gometalinter can't work in go mod environment.
+go-mod-vendor:
+	@echo "Begin to vendor go mod dependency"
+	@go mod vendor
+.PHONY: go-mod-tidy
```

Comment:
go-mod-vendor？

Reply:
Yes, see this issue alecthomas/gometalinter#562. Currently gometalinter cannot work in go mod env without vendor. So I have to add this during ci when checking gometalinter.

Reply:
Done.